src/CMakeLists.txt

```diff
@@ -1 +1,63 @@
-/opt/ros/kinetic/share/catkin/cmake/toplevel.cmake
\ No newline at end of file
+# toplevel CMakeLists.txt for a catkin workspace
```

Comment:
Why did the top level cmake change? that's a generated file at repository setup.

Reply:
Not a required change, just curious

Reply:
It doesn't build on my computer when it's just that path.  The file at that path exists, but my catkin couldn't parse the CMakeLists for some reason.

Reply:
Everyone else I've asked can build it just fine, including the Jenkins server.  I'm not sure why my system is different.